.gitignore

```diff
@@ -0,0 +1,2 @@
+.vscode/
+.DS_Store
```

Comment:
isn't it missing more stuff? like node_modules etc

Reply:
I felt the same way, but as there is no `package.json`, there isn't much to add. On the example project there is an extensive `.gitgnore`, but looks exaggerated and unnecessary to me. What do you say?

the file: https://github.com/vtex-apps/graphql-interface-app-example/blob/c78a9a1a2a4f197dccfd302eb36088beb97a4eb7/graphql-interface-app/.gitignore

Reply:
I'd rather add it now because we might want something like prettier to auto format the graphql queries